poetry/config/config.py

```diff
@@ -20,11 +20,11 @@
 _NOT_SET = object()
 
 
-def boolean_validator(val):
+def boolean_validator(val):  # type: (str) -> bool
```

Comment:
why don't use type annptations?

Reply:
You mean `def boolean_validator(val: str) -> bool:` instead? Because this is not supported by python2. And we haven't dropped python2 support until now. Yes the master branch is for the next feature release, which will drop python2 support. But I plan to port these changes to the 1.1 branch as well and switch to the type annotation syntax after the next feature release. Otherwise the code base between the to branches becomes to different and it becomes more difficult to port fixes between them.